### PR TITLE
Added 2 new anticipated sensors

### PR DESCRIPTION
**Pull request by mkanet**
I added 2 new sensors called:

- sensor.trakt_anticipated_shows
- sensor.trakt_anticipated_movies

Example configuration to enable the sensors above:

```
trakt_tv:
  language: en
  timezone: America/Los_Angeles
  sensors:
    anticipated:
      movie:
        max_medias: 10
      show:
        max_medias: 10
```
Example configuration will additionally exclude collected items:

```
trakt_tv:
  language: en
  timezone: America/Los_Angeles
  sensors:
    anticipated:
      movie:
        exclude_collected: true
        max_medias: 10
      show:
        exclude_collected: true
        max_medias: 10
```

### CODE REVIEW

**Comment by mkanet** [comment]
@dylandoamaral Are you still able to merge this into your latest code?  I spent a lot of time on this PR.  I am hoping all you have to do is merge it.  My latest code includes the changes in this PR including the `Summary` and `Trailer` attributes.  Let me know how you want to proceed.  Hopefully, it won't be too difficult to move forward with this.

**Comment by dylandoamaral** [comment]
Yeah I'll check when I have time !

**Comment by mkanet** [comment]
> Yeah I'll check when I have time !

I can't wait!!!  BTW: I have been personally using these new features every day on my personal Home Assistant setup ever since I submitted my PRs.  These features are easily my most favorite/useful features on this Trakt integration currently.

File: custom_components/trakt_tv/apis/trakt.py [hunk 1]
```diff
@@ -405,6 +406,50 @@ async def fetch_stats(self):
...
+                for kind in kinds
+            ]
+        )
+
+        res = {}
+
+        for trakt_kind, raw_medias in zip(kinds, data):
+            if raw_medias is not None:
+                medias = [
+                    trakt_kind.value.model.from_trakt(media["movie" if trakt_kind == TraktKind.ANTICIPATED_MOVIE else "show"]) for media in raw_medias
```
Comment (dylandoamaral) [inline custom_components/trakt_tv/apis/trakt.py:444]: trakt_kind.identifier instead of if else.

**Review by dylandoamaral — COMMENTED** [review]
This change + the README and we are good to go for a release, good job !

**Comment by mkanet** [comment]
@dylandoamaral thank you so much for this!  Please see latest commit which includes readme update as well as the small code change you suggested.  Let me know if that's all you need to merge and update the official repo.

**Comment by dylandoamaral** [comment]
Perfect thank you again for your time !

**Comment by mkanet** [comment]
You're welcome. I'm looking forward to seeing this official update!

**Pull request merged**